client/sass/_colors.sass

```diff
@@ -13,3 +13,8 @@ $color-warning-light: lighten($color-warning, 5%)
 $color-success: #8ac159
 $color-success-dark: darken($color-success, 5%)
 $color-success-light: lighten($color-success, 5%)
+
+$white: #FFF
```

Comment:
It might be my OCD but I feel like we should write these all down lowercase OR uppercase, but should not mix cases.

Reply:
Agreed, all lowercase now